src/main/resources/datamapper.properties

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# WSO2 Inc. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+# ------------------------------------------
+# Data Mapper Compiler Plugin Error Messages
+# ------------------------------------------
+
+error.invalid.attribute.name=\
+  type ''{0}'' does not have an attribute named ''{1}''
+
+error.invalid.attribute.count=\
+  invalid attribute count: expected ''{0}'', found ''{1}''
+
+error.invalid.json.content=\
+  invalid json content: ''{0}''
```

Comment:
json -> JSON?

Reply:
Fixed in [3d3608d](https://github.com/ballerina-platform/module-ballerinax-datamapper/pull/36/commits/3d3608d555b8b40bb1d0fb9cd95d9099af3a80d2)